tests/utils/html-element.js

```diff
@@ -0,0 +1,13 @@
+export function setUpHTMLElement(options) {
```

Comment:
I'm not sure this will be useful for anything else than testing the scroll functionality in dropdown-search, I'd keep it in the spec.js file then, maybe rename it to `mockHtmlElementPosition`. I'm not sure how the teardown of this mock will look like.